source/TradingPanel.cpp

```diff
@@ -128,7 +128,7 @@ void TradingPanel::Draw()
 		for(const auto &it : player.Cargo().Outfits())
 			if(it.second)
 			{
-				bool isHarvested = (it.first->Get("installable") < 0.);
+				bool isHarvested = (it.first->Get("minable") > 0.);
 				(isHarvested ? hasHarvested : hasOutfits) = true;
 			}
 		sellOutfits = (hasOutfits && !hasHarvested);
```

Comment:
I'm not necessarily sure about this change. In the spirit of your PR (changing installable checks to minable checks) yes, it does make sense, but we should consider the end goal w.r.t. "sell outfits" behavior.

Right now the default behavior when you have installable outfits, commodities, and harvestable items, you can use "Sell All" to sell all the "unusable" items, and then you can sell again to sell the usable outfits that remained.

Even if we change how we identify something that can be found in an asteroid payload, we probably still want to keep the "sell unusable things when selling commodities" behavior.

(In the future we should further limit the scope of selling uninstallable outfits to avoid selling "junk" that an active mission uses.)

Reply:
What would be a way to resolve that, then? A third `junk` attribute, which determines if an outfit gets sold in the bulk "Sell All" action? Then otherwise, it would have to be sold in "Sell Outfits" or manually?

Reply:
I'm suggesting that maybe this change isn't correct in terms of having the correct "sell junk" behavior, and that the original code may be more correct.

I don't think that reserving another attribute is warranted.

The future comment would be resolved by comparing a minable outfit to be sold against the minable outfits needed by currently active missions. (Since that isn't a thing yet, it can't be resolved yet either.)

Reply:
I was leaving it open in case some outfit designer wanted to have an uninstallable outfit that wasn't necessarily a bulk commodity, like asteroid minerals or illegal commodities, but it's tough to think of what those would be. Probably for the best to include it in this category, and then have a potential future safety net with missions as an option, yeah.

In that case, what would we rename "harvested materials" to in the trade panel? "Junk" might be a bit slang-y, but it feels like that's been adopted as a less meta term for "vendor trash" in games. "Sellable goods" could work, but feels a bit awkward. Anyone have any other thoughts?

Reply:
Right now, asteroids are defined as uninstallable, which seems like a hacky way to do it. We should keep it though, for retrocompatibility.
Asteroids are simply considered as "Special", a junk category that handles the left overs, I think they should instead be defined as a new category on their own: minable, thus showing up differently in the outfitter panel.

Then these new smuggling goods **need** a category of their own right, instead of just using the special category, which is already filled with licenses so much I never bother to check it.
On top of that, it does not make sense for smuggling goods to be shown, and sold in the same way as asteroids, you have taken way more risk to come here to sell them, so you'll better go check if it's for the right price, this isn't some random selling 500 food kind of thing. For me the sell all button is "get rid of junk" button I always press, not "sell 1M worth of smuggling goods" which I just had a hard time getting here.

TLDR:
add 2 new categories : smuggling and minable (I woudn't mind minable staying in special category, but smuggling goods don't belong there), those 2 things should be treated separately in the spaceport or you'll just never see them, in the bottom of the list with all those licenses

Reply:
if you don't agree with new categories then at least show how much money you made by selling the goods as you sell them not as you take off and can't take them back.